Issues with module-info after upgrading to 6.9.0

### Describe the bug

We are previously on version 6.7.2 and we are trying to uptake 6.9.0 

But our builds are failing with below error 

```
[ERROR] error: module knative.client reads package io.fabric8.kubernetes.api.builder from both kubernetes.model.core and kubernetes.model.common
[ERROR] error: module knative.model reads package io.fabric8.kubernetes.api.builder from both kubernetes.model.core and kubernetes.model.common
[ERROR] error: module com.fasterxml.jackson.databind reads package io.fabric8.kubernetes.api.builder from both kubernetes.model.core and kubernetes.model.common
[ERROR] error: module org.reflections reads package io.fabric8.kubernetes.api.builder from both kubernetes.model.core and kubernetes.model.common
[ERROR] error: module org.checkerframework.checker.qual reads package io.fabric8.kubernetes.api.builder from both kubernetes.model.core and kubernetes.model.common
[ERROR] error: module com.fasterxml.jackson.core reads package io.fabric8.kubernetes.api.builder from both kubernetes.model.core and kubernetes.model.common
[ERROR] error: module kubernetes.model.common reads package io.fabric8.kubernetes.api.builder from both kubernetes.model.core and kubernetes.model.common
[ERROR] error: module kubernetes.client.api reads package io.fabric8.kubernetes.api.builder from both kubernetes.model.common and kubernetes.model.core
[ERROR] error: module kubernetes.model.core reads package io.fabric8.kubernetes.api.builder from both kubernetes.model.core and kubernetes.model.common
```



This is our module-info.java 

```
module com.demo {
    requires com.fasterxml.jackson.annotation;
    requires kubernetes.model.core;
    requires static org.checkerframework.checker.qual;
    requires org.reflections;
    requires java.logging;
    requires com.fasterxml.jackson.databind;
    requires org.graalvm.sdk;
    requires knative.model;
    requires knative.client;
    requires kubernetes.model.common;
    requires kubernetes.client.api;
}
```

### Fabric8 Kubernetes Client version

6.9.0

### Steps to reproduce

1. Just build with latest fabric8 client 

### Expected behavior

Build should be successful

### Runtime

other (please specify in additional context)

### Kubernetes API Server version

1.25.3@latest

### Environment

macOS

### Fabric8 Kubernetes Client Logs

_No response_

### Additional context

_No response_

--- a/kubernetes-model-generator/kubernetes-model-core/pom.xml
+++ b/kubernetes-model-generator/kubernetes-model-core/pom.xml
@@ -99,7 +99,6 @@
     </osgi.import>
     <osgi.export>
       io.fabric8.kubernetes.api,
-      io.fabric8.kubernetes.api.builder,
       io.fabric8.kubernetes.api.model**,
     </osgi.export>
     <osgi.private>

